### DISCUSSION TITLE
Any possibilities to use Lazy Loading support with CLI ?

### DISCUSSION THREAD

**Discussion by laurent-d** [percy/cli#461] [Support] [answered]
Hi

After reading this doc : [https://docs.percy.io/docs/capturing-lazy-loading-images ](https://docs.percy.io/docs/capturing-lazy-loading-images )
I discovered that is possible to take account of the lazy loading of images in visual testing.
But it is based on the deprecated Percy Script not the CLI usage : somebody have an idea about it ?
Is it doable ?

Cheers

**Comment by Robdel12** [comment, marked as answer]
Hey @laurent-d! The linked doc provides examples for a few SDKs -- it's the same for any SDK. The main take away is you'll have to scroll the page in your test suite prior to calling `percySnapshot` (so the DOM changes are present when we capture the DOM). 

I update the doc to remove references to PercyScript since it's deprecated.

**Reply by laurent-d** [reply]
thanks @Robdel12 I will search in this way
